### PR TITLE
Flathub Search Command

### LINKED ISSUE #1072: Flathub Search Command
> *Flathub* is the home of hundreds of apps that can be easily installed on any Linux distribution. Browse the apps online, from your app center or the command line.

https://flathub.org/home

`s!flathub <search query>`

### PR DESCRIPTION
**Pull request by DraftProducts**
Hey, here is the flathub command which I just created, hope that will be ok 👍

Closes #1072 

image

### CODE REVIEW

File: src/commands/misc/flathub.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const moment = require('moment')
+
+module.exports = class FlatHubCommand extends SearchCommand {
+  constructor (client) {
+    super({
+      name: 'flathub',
+      aliases: ['flath'],
+      requirements: { apis: ['flathub'] },
+      embedColor: Constants.FLATHUB_COLOR,
+      embedLogoURL: 'https://flathub.org/assets/themes/flathub/flathub-logo.png'
```
Comment (almeidx) [inline src/commands/misc/flathub.js:13]: Please refrain from using third-party websites. At best, [upload the image to Imgur](https://imgur.com)

**Review by almeidx — CHANGES_REQUESTED** [review]

File: src/commands/misc/flathub.js (hunk 1 above)
Comment (DraftProducts) [reply src/commands/misc/flathub.js:13]: Do you want that i do a pull request for all third-party links ?
There are a lot of links coming from a others websites

**Review by almeidx — COMMENTED** [review]
Also, could you use [axios](https://www.npmjs.com/package/axios) instead of node-fetch? As per #1096

File: src/commands/misc/flathub.js (hunk 1 above)
Comment (almeidx) [reply src/commands/misc/flathub.js:13]: I meant for static image URLs only. But if you were talking about this as well, then sure!

File: src/apis/FlatHub.js [hunk 2]
```diff
@@ -11,14 +11,17 @@ module.exports = class FlatHub extends APIWrapper {
   }
 
   async list () {
-    return fetch(API_URL).then(r => r.json())
+    const res = await axios({ baseURL: API_URL });
```
Comment (almeidx) [inline src/apis/FlatHub.js]:
```suggestion
    const res = await axios(API_URL);
```

**Review by almeidx — CHANGES_REQUESTED** [review]

File: src/apis/FlatHub.js (hunk 2 above)
Comment (DraftProducts) [reply src/apis/FlatHub.js]: Ok

**Review by almeidx — APPROVED** [review]

**Review by Doges — APPROVED** [review]
LGTM

**Comment by DraftProducts** [comment]
Nice :)

**Pull request merged**